docs/concepts/07_dip/04_dapp_developer.md

```diff
@@ -0,0 +1,11 @@
+---
+id: dapp-developer
+title: Dapp developer
+---
+
+For Dapp developers, the Decentralized Identity Provider (DIP) has two resources to help you build applications that take advantage of DIP.
+
+- The DIP SDK:
+- The KILT DID utilities library
```

Comment:
Again, we should not advertise this. It is maintained on a best effort basis by me, and not tested at all. DIP SDK will be more thoroughly tested and guaranteed to work.